### PR TITLE
Add hook for ScreenshotReady event to NotificationScript2

### PR DESCRIPTION
**Pull request by tiffany352**
This flag removes both the old internet explorer window and the old ShowMessage() based notifications for screenshots, replacing them with a corescripts notification.

### CODE REVIEW

File: CoreScriptsRoot/CoreScripts/NotificationScript2.lua [hunk 1]
```diff
@@ -40,6 +40,9 @@ local newNotificationPath = getNewNotificationPathSuccess and newNotificationPat
 local getTenFootBadgeNotifications, tenFootBadgeNotificationsValue = pcall(function() return settings():GetFFlag("TenFootBadgeNotifications") end)
 local tenFootBadgeNotifications = getTenFootBadgeNotifications and tenFootBadgeNotificationsValue
 
+local disableScreenshotPopup, disableScreenshotPopupValue = pcall(function() return settings():GetFFlag("DisableScreenshotPopup") end)
+local disableScreenshotPopup = disableScreenshotPopup and disableScreenshotPopupValue
```
Comment (hockeychaos) [inline CoreScriptsRoot/CoreScripts/NotificationScript2.lua]: is a little weird to declare the same local variable as the previous line here.

File: CoreScriptsRoot/CoreScripts/NotificationScript2.lua [hunk 2]
```diff
@@ -721,6 +724,22 @@ end
 
 end
 
+if disableScreenshotPopup then
+	game.ScreenshotReady:Connect(function(path)
```
Comment (hockeychaos) [inline CoreScriptsRoot/CoreScripts/NotificationScript2.lua]: is this a new event on game? should we pcall to be safe?

Comment (tiffany352) [inline CoreScriptsRoot/CoreScripts/NotificationScript2.lua]: It's a new event, but it's behind an fflag, so it's safe.

File: CoreScriptsRoot/CoreScripts/NotificationScript2.lua [hunk 3]
```diff
@@ -721,6 +724,22 @@ end
...
+		sendNotificationInfo {
+			Title = "Screenshot Ready",
```
Comment (hockeychaos) [inline CoreScriptsRoot/CoreScripts/NotificationScript2.lua]: sorry, one minor thing, lets rename to "Screenshot Taken"

**Pull request merged**